community/kernel/src/main/java/org/neo4j/graphdb/impl/notification/NotificationCode.java

```diff
@@ -79,8 +79,8 @@
         Status.Statement.DynamicPropertyWarning,
         "Using a dynamic property makes it impossible to use an index lookup for this query"
     ),
-    BARE_NODE_SYNTAX_DEPRECATED(
-        SeverityLevel.WARNING,
+    BARE_NODE_SYNTAX_DEPRECATED( // This notification is no longer produced by current Cypher compilers
+        SeverityLevel.WARNING,   // but it is left here for backwards compatibility.
```

Comment:
This means that the `kernel` module will keep backwards compatibility to `cypher-compiler` modules of previous Neo4j versions? Do we have any such inter-modular compatibility policies?